pkg/fileserver/server_windows.go

```diff
@@ -1,3 +1,5 @@
+//go:build windows
+// +build windows
```

Comment:
this file is already called _windows.go so it should already be only compiled on windows anyway so this tag should not be necessary